server/ERNI.PBA.Server.Host/Utils/TeamBudgetExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using ERNI.PBA.Server.DataAccess.Model;
+
+namespace ERNI.PBA.Server.Host.Utils
+{
+    public static class TeamBudgetExtensions
```

Comment:
I believe this is in the end more code than the generic solution (because of this mapping and the necessity of the TeamBudget class), but it works the same, so it's ok.